doc/userguide/install.rst

```diff
@@ -75,40 +75,100 @@ For full features, also add::
 
 Rust support::
 
-  rustc, cargo
+  rustc, cargo, cbindgen
```

Comment:
I think this line is a leftover from when Rust support was optional, which it no longer is. So I wonder if this can be integrated into the required tools.

Users installing from release packages don't need cbindgen, but it doesn't hurt either.

Reply:
I personally don't find the listed dependencies particularly helpful so I removed the whole section of `Dependencies` and moved the Rust part to the end of installation instructions - so that 95% of users should be able to install it without issues as the distros will have sufficient Rust version available. Those 5% of users will find their remedy at the end, linking Rust's official webpage as a place to install the newer Rust version.